src/Microsoft.Graph.Core/Requests/GraphClientFactory.cs

```diff
@@ -112,7 +112,7 @@ public static IList<DelegatingHandler> CreateDefaultHandlers(GraphClientOptions
         {
             return new List<DelegatingHandler> {
                 new GraphTelemetryHandler(graphClientOptions),
```

Comment:
is there any specific reason here why we don't call the kiota factory to get the default handlers for kiota and just add the telemetry on top of that?

Reply:
Thanks for calling this out. 
This was a means to control the handler order due to do some remaining work in https://github.com/microsoft/kiota/issues/598.

I've now incorporated proposals made from https://github.com/microsoftgraph/msgraph-sdk-design/pull/75 and added the call to the kiota factory.